packages/sinuous/observable/src/observable.js

```diff
@@ -172,9 +170,16 @@ function computed(observer, value) {
     });
 
     // If any children were marked as fresh remove them from the run lists.
-    const allChildren = getChildrenDeep(update._children);
-    allChildren.forEach(removeFreshChildren);
-
+    let curr;
+    const queue = [].concat(update.children);
```

Comment:
just noticed, missing the underscore here. `update._children`

Reply:
WHOA. Huh sorry I missed that converting from a TS codebase back to JS... I'm too used to the editor catching that :(